index.html

```diff
@@ -11,55 +11,56 @@
     <div class="jumbotron">
       <div class="container">
         <img src="/assets/images/trucol-logo.svg" alt="TruCol logo">
-        <p><i>TruCol</i> is a free and open-source reference guide that explains how to use Markdown, the simple and easy-to-use markup language you can use to format virtually any document.</p>
+        <p><i>TruCol</i> is a decentralised collaboration protocol, which promotes <em>inclusive</em>, <em>fair</em> and <em>accessible</em> work, by
+          enabling developers to participate in the market regardless of their circumstances.</p>
```

Comment:
Perhaps the accuracy of this statement can be improved by changing `circumstances` to `geosocial circumstances, gender or ethnicity`.

Reply:
I have changed it to your suggestion